integrations/spiral/README.md

```diff
@@ -124,49 +143,78 @@ return [
 ];
 ```
 
-The default engine is available as `Engine`:
+## Usage
```

Comment:
basic example should be keept in all integrations the same way and as small as possible.

This part would be more some kind of a cookbook how to index `Spiral` `Models` into the index.

Reply:
It's not about index spiral models, there is an abstract code, which shows only more understandable example, what to do with $engine object.